protected/models/Patient.php

```diff
@@ -2089,7 +2089,7 @@ public function dateOfBirthRangeValidator($attribute, $params)
      * @param $last_name string Last name.
      * @param $dob string Date of Birth (DD/MM/YYYY).
      * @param $id int ID of the current patient record.
-     * @return array|Patient[] The list of patients who have similar names and the same date of birth.
+     * @return array The list of patients who have similar names and the same date of birth, or the invalid patient model.
```

Comment:
I know you mentioned this earlier, but I didn't realise that you were actually mixing return types, that feels a bit messy. I'm not sure of a good way around it though.

Reply:
This is the only way I could think of to make this work without a significant refactor.